ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/BatchingParallelIngestionPipe.scala

```diff
@@ -42,9 +40,36 @@ object BatchingParallelIngestionPipe {
       .mapAsync(ingestingParallelism)(ingester)
       // Stage 6: Preparing data sequentially for throttled mutations in database (tracking the ledger-end, corresponding sequential event ids and latest-at-the-time configurations)
       .conflate(tailer)
-      .throttle(tailingRateLimitPerSecond, FiniteDuration(1, "seconds"))
       // Stage 7: Updating ledger-end and related data in database (this stage completion demarcates the consistent point-in-time)
       .mapAsync(1)(ingestTail)
       .map(_ => ())
 
+  /** Forms dynamically-sized batches based on downstream backpressure.
```

Comment:
would you consider separating this batchN into its own object? I could imagine we might even want to reuse this other places

Reply:
sure, good idea